python/lsst/meas/algorithms/astrometrySourceSelector.py

```diff
@@ -43,6 +43,18 @@ class AstrometrySourceSelectorConfig(BaseSourceSelectorConfig):
         "(in the flux specified by sourceFluxType); <= 0 for no limit",
         default=10,
     )
+    bad_flags = pexConfig.ListField(
```

Comment:
Huh, I didn't think I touched this file. This was an unintentional change. I'll revert it back to the version that is on master.